client/assets/stylesheets/shared/_reset.scss

```diff
@@ -123,6 +123,16 @@ a:active {
 a img {
 	border: 0;
 }
+button {
+	-webkit-appearance: none;
+	appearance: none;
+	background: transparent;
+	border: none;
+	font-size: 14px;
```

Comment:
Not so sure about this rule, value seems too specific for a reset.

Reply:
Agreed. What we really want to is to override the user agent default font for buttons and set `font-size: inherit`. The `font-size: 14px` needs to be moved to the component-specific `.button` class.

This will break a few places (like the Login page's "Change username" button), because they will then have 15px size instead of 14px. Needs to be fixed. But the change is worth it.

Reply:
Changed this to `inherit`.

I'm looking at the "Change username" button and it has `font-size:14px` because it inherits that from the surrounding `label` element.

<img width="1504" alt="image" src="https://user-images.githubusercontent.com/87168/67563501-7b8bf380-f729-11e9-9247-9160eb702d14.png">

There are however annoyingly many `<button>` elements without `.button` class to test...

What do you think if in this PR we'd add it to base styles at `client/assets/stylesheets/_main.scss`?

Reply:
> What do you think if in this PR we'd add it to base styles 

You mean doing the `font-inherit` change? Yes, go for it. Most usages of a naked `<button>` are likely to be elements with no text (4 of 5 examples I found earlier today were like that), and if anything gets 15px instead of 14px, we can iron it out in later PRs.

Reply:
I meant adding `button { font-size:14px; }` in the main, just to have it somewhere so to ensure everything looks like it does now.

Reply:
The goal of `button { font-size: 14px; }` in `main` would be to remove it from `reset`, where it doesn't belong? And, unlike the risky `.button { font-size: 14px }`, making sure that nothing is broken?

If you mean the following, I say yes 🙂 
- put `button { font-size: inherit }` into the reset
- put `button { font-size: 14px }` into main
- put `.button { font-size: 14px }` into `Button` component style

That makes the reset right, the `Button` component right, and we can remove the main style in another PR after checking everything.

Reply:
> If you mean the following, I say yes 🙂
> 
>     put button { font-size: inherit } into the reset
>     put button { font-size: 14px } into main
>     put .button { font-size: 14px } into Button component style

My thoughts exactly. 👍  Done!